front/app/containers/MobileNavbar/FullMobileNavMenu.tsx

```diff
@@ -0,0 +1,256 @@
+import React, { useEffect } from 'react';
```

Comment:
This is the menu that opens when you click the new 'More' button in the mobile navigation.